packages/quantic/force-app/main/default/lwc/quanticTimeframeFacet/__tests__/quanticTimeframeFacet.test.js

```diff
@@ -127,6 +191,31 @@ function mockSuccessfulHeadlessInitialization() {
   };
 }
 
+function mockErroneousHeadlessInitialization() {
+  // @ts-ignore
+  mockHeadlessLoader.initializeWithHeadless = (element) => {
+    if (element instanceof QuanticTimeframeFacet) {
+      element.setInitializationError();
+    }
+  };
+}
+
+async function submitFacetSearchForm(element) {
+  const form = element.shadowRoot.querySelector(selectors.datePickerForm);
+  form.dispatchEvent(
+    new CustomEvent('submit', {bubbles: true, composed: true})
+  );
+  await flushPromises();
+}
+
+async function mockInputReportValidity(element) {
```

Comment:
What is async in this function?

Reply:
will remove, that was just a copy paste error.